src/Lambertian.cpp

```diff
@@ -6,6 +6,9 @@ bool Lambertian::shaderProgramCompiled = false;
 unsigned int Lambertian::shaderProgram = 0;
 unsigned int Lambertian::vertexShader = 0;
 unsigned int Lambertian::fragmentShader = 0;
+const glm::vec3 Lambertian::xAxis(1,0,0);
```

Comment:
Minor nitpick: can y'all add some spaces here so it's `const glm::vec3 Lambertian::xAxis(1, 0, 0);` instead?